### PR TITLE
Np 3359 get by identifier

### PR DESCRIPTION
**Pull request by svenngjeto**

### CODE REVIEW

File: cristin-organization/src/test/resources/unit_18_90_0_0.json
Comment (torarnet) [inline cristin-organization/src/test/resources/unit_18_90_0_0.json:259]: Something seems familiar

File: cristin-commons/src/main/java/no/unit/nva/model/Organization.java
Comment (torarnet) [inline cristin-commons/src/main/java/no/unit/nva/model/Organization.java:29]: Is this a context uri or a regular uri?

File: cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java
Comment (torarnet) [inline cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java:53]: Is `getNvaApiId` a correct name for somethings which add `ORGANIZATION_PATH` to path? Maybe add `String type` to parameters?

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (torarnet) [inline cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:63]: This can be replaced by the method you created in `UriUtils` after adding parameter `String type`

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (torarnet) [inline cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:32]: `CristinQueryHandler` is meant for queries, not fetch. Im working on this in another branch, where i can reuse a lot of things from `CristinHandler` and `CristinQueryHandler` to reduce duplicate code

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (torarnet) [inline cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:56]: Can this be moved to `UriUtils` and be reused? Or should we keep it as is?

Comment (torarnet) [reply cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:56]: I mean the building of the Cristin uri

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/HttpExecutorImpl.java
Comment (torarnet) [inline cristin-organization/src/main/java/no/unit/nva/cristin/organization/HttpExecutorImpl.java:48]: What does `super()` do? The class does not extend anything...

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (torarnet) [reply cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:32]: I see you have extended it in your Query class as well. This reduces the scope of my other branch, as we have done exactly the same. Maybe I dont need to do this refactoring in a separate PR but include it in the one im currently working on as there now is less lines of code in my branch to merge

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/QueryCristinOrganizationHandler.java
Comment (torarnet) [inline cristin-organization/src/main/java/no/unit/nva/cristin/organization/QueryCristinOrganizationHandler.java:31]: Do you need all these constants? I dont think they are being used anymore?

File: cristin-organization/src/test/java/no/unit/nva/cristin/organization/QueryCristinOrganizationHandlerTest.java
Comment (torarnet) [inline cristin-organization/src/test/java/no/unit/nva/cristin/organization/QueryCristinOrganizationHandlerTest.java:66]: Use the method in `UriUtils`

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/HttpExecutorImpl.java
Comment (torarnet) [inline cristin-organization/src/main/java/no/unit/nva/cristin/organization/HttpExecutorImpl.java:109]: How does this work with UTF-8?

File: cristin-organization/src/test/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandlerTest.java
Comment (torarnet) [inline cristin-organization/src/test/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandlerTest.java:187]: Reuse method in `UriUtils`

**Review by torarnet — CHANGES_REQUESTED** [review]
Have some comments I want you to look at before approval

File: cristin-commons/src/main/java/no/unit/nva/model/Organization.java
Comment (svenngjeto) [reply cristin-commons/src/main/java/no/unit/nva/model/Organization.java:29]: It's a contexturi waiting to be determined... Just a placeholder for now

File: cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java
Comment (svenngjeto) [reply cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java:53]: It constructs a new uri against the NVA api, and adds the string containing four numbers (identifier) -> id

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (svenngjeto) [reply cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:63]: The context wil changed to something completely different I guess

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (svenngjeto) [reply cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:32]: It'is prepared for the furure and APIGatewayHandler is grandfather anyway...

File: cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java
Comment (svenngjeto) [reply cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java:56]: Added to UriUtils

File: cristin-organization/src/test/java/no/unit/nva/cristin/organization/QueryCristinOrganizationHandlerTest.java
Comment (torarnet) [inline cristin-organization/src/test/java/no/unit/nva/cristin/organization/QueryCristinOrganizationHandlerTest.java:71]: Is this missing something? Maybe query params?

**Review by torarnet — APPROVED** [review]
Approved but needs review from more people

**Review by brinxmat — APPROVED** [review]

**Pull request merged**
